chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/surveillance/AddDocumentActionPermissions.java

```diff
@@ -93,6 +98,15 @@ private boolean isNonconformityForRemovedCriteria(SurveillanceNonconformityEntit
                 && nonconformity.getCertificationCriterionEntity().getRemoved().booleanValue();
     }
 
+    private boolean isNonconformityForRemovedRequirement(SurveillanceNonconformityEntity nonconformity) {
+        if (!ff4j.check(FeatureList.EFFECTIVE_RULE_DATE_PLUS_ONE_WEEK)) {
+            return false;
+        }
+
+        return nonconformity != null
+                && nonconformity.getType().equalsIgnoreCase("170.523 (k)(2)");
```

Comment:
I am guilty of not always doing this but can we make this string a constant somewhere? I see it referenced in a number of the files throughout the PR.

Reply:
Where would you suggest it go? The string is referenced from both domain objects and reviewers, so I don't think it could be in a base class somewhere they both extend from. Do we have a "constants" file somewhere that would make sense to include it?

Reply:
It looks like we already have it in RequirementTypeEnum

Reply:
and another enum NonconformityType